python/lsst/obs/decam/instrument.py

```diff
@@ -0,0 +1,86 @@
+# This file is part of obs_decam.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (http://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Butler instrument description for the Dark Energy Camera.
+"""
+
+__all__ = ("DarkEnergyCamera",)
+
+import os
+
+from lsst.afw.cameraGeom import makeCameraFromPath, CameraConfig
+from lsst.daf.butler.instrument import Instrument
+from lsst.obs.decam.decamFilters import DECAM_FILTER_DEFINITIONS
+
+from lsst.utils import getPackageDir
+
+
+class DarkEnergyCamera(Instrument):
+    """Gen3 Butler specialization class for the DECam instrument.
+    """
+    filterDefinitions = DECAM_FILTER_DEFINITIONS
+
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
+        packageDir = getPackageDir("obs_decam")
+        self.configPaths = [os.path.join(packageDir, "config")]
+
+    @classmethod
+    def getName(cls):
+        return "DECam"
```

Comment:
Another place for the camera name (in addition to that coming from the `ObservationInfo` and the config)

Reply:
I'm not sure what you meant by this comment: should I be pulling this string from somewhere else?

Reply:
I am trying to say we should be very careful about how many places we name the instrument. There are at least three places that possibly have to agree. I'm okay with punting on this in the hope we have a future plan.